snapshot-labs/snapshot
# fix: Remove Hal plugin

### Summary

[Remove hal plugin as it is deprecated](https://discord.com/channels/955773041898573854/1030772384308932608/1275599987102973974)

### How to test

1. go to http://localhost:8080/#/stgdao.eth/proposal/0x38d9496b04f51a376a2c4979dea6c499edd5c335e98aed0e05f4070c2a7d5c5d
2. you should not see hal plugin anymore
3. In your space settings, you should not see this plugin in plugins list
